### ISSUE TITLE
Connection string syntax not compatible with documentation.

### ISSUE THREAD

**Issue by forejtv** [EventStore/EventStore-Client-NodeJS#123]
The documentation at https://developers.eventstore.com/clients/grpc/getting-started/#connection-details makes it possible to generate a connection string. For example, by picking "Specify manually", "Single node", "Insecure" and "localhost:2113", the connection string will be `esdb://localhost:2113?Tls=false`.

On the subsequent page, the NodeJS code for creating the client is:
```javascript
const client = EventStoreDBClient.connectionString("esdb://localhost:2113?Tls=false");
```
When executing this code, a warning is printed saying
```
Unknown option key "Tls", setting will be ignored.
```
and the connection to the database fails. If I change `Tls` to `tls` in the connection string (lowercase `t`), which is consistent to what the parsing code expects at https://github.com/EventStore/EventStore-Client-NodeJS/blob/master/src/Client/parseConnectionString.ts#L265, the connection works fine.

Tested on: `0.0.0-alpha.12`

**Comment by Eadword** [comment]
This appears to have been resolved in Alpha 13 as I am able to connect using `esdb://eventstore:2113?Tls=false` within my docker configuration.

**Comment by George-Payne** [comment]
Hi Vojtěch,

This was fixed by #109 and is available in `0.0.0-alpha.13`
